### ISSUE TITLE
[BUG] Deprecation Warning: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

### ISSUE THREAD

**Issue by fritzyfinance** [ColorlibHQ/AdminLTE#4015]
Bootstrap 4.6.1 is out and solved this depreciation warning.

Please find the list of this warning in admin lte v3.1.0

```
⠦ building... [Package /assets/vendor.js]Deprecation Warning: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($nav-link-padding-x, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
141 │ $sidebar-user-image-width: $sidebar-nav-icon-width + ($nav-link-padding-x / 2) !default;
    │                                                       ^^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/admin-lte/build/scss/_variables.scss 141:55                                                      @import
    node_modules/admin-lte/build/scss/_adminlte.raw.scss 21:9                                                     @import
    /tmp/broccoli-3872866n6fuv7MpaJUp/out-296-broccoli_merge_trees_full_application/app/styles/app.scss#sass 2:9  root stylesheet

Deprecation Warning: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(-$sidebar-padding-x, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
507 │   margin: (-$sidebar-padding-x / 2) (-$sidebar-padding-x) 0;
    │            ^^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/admin-lte/build/scss/_main-sidebar.scss 507:12                                                   @import
    node_modules/admin-lte/build/scss/parts/_core.scss 11:9                                                       @import
    node_modules/admin-lte/build/scss/_adminlte.raw.scss 25:9                                                     @import
    /tmp/broccoli-3872866n6fuv7MpaJUp/out-296-broccoli_merge_trees_full_application/app/styles/app.scss#sass 2:9  root stylesheet

Deprecation Warning: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(-$sidebar-padding-x, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
672 │   margin: (-$sidebar-padding-x / 2) (-$sidebar-padding-x) 0;
    │            ^^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/admin-lte/build/scss/_main-sidebar.scss 672:12                                                   @import
    node_modules/admin-lte/build/scss/parts/_core.scss 11:9                                                       @import
    node_modules/admin-lte/build/scss/_adminlte.raw.scss 25:9                                                     @import
    /tmp/broccoli-3872866n6fuv7MpaJUp/out-296-broccoli_merge_trees_full_application/app/styles/app.scss#sass 2:9  root stylesheet

Deprecation Warning: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($nav-link-padding-y, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
885 │     padding-top: ($nav-link-padding-y / 2);
    │                   ^^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/admin-lte/build/scss/_main-sidebar.scss 885:19                                                   @import
    node_modules/admin-lte/build/scss/parts/_core.scss 11:9                                                       @import
    node_modules/admin-lte/build/scss/_adminlte.raw.scss 25:9                                                     @import
    /tmp/broccoli-3872866n6fuv7MpaJUp/out-296-broccoli_merge_trees_full_application/app/styles/app.scss#sass 2:9  root stylesheet

Deprecation Warning: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($nav-link-padding-y, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
886 │     padding-bottom: ($nav-link-padding-y / 2);
    │                      ^^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/admin-lte/build/scss/_main-sidebar.scss 886:22                                                   @import
    node_modules/admin-lte/build/scss/parts/_core.scss 11:9                                                       @import
    node_modules/admin-lte/build/scss/_adminlte.raw.scss 25:9                                                     @import
    /tmp/broccoli-3872866n6fuv7MpaJUp/out-296-broccoli_merge_trees_full_application/app/styles/app.scss#sass 2:9  root stylesheet

Warning: 12 repetitive deprecation warnings omitted.
```

**Comment by danny007in** [comment]
update to latest version of dart sass

```console
npm i sass@latest -D
```

**Comment by forxer** [comment]
This is not a problem with the version of Dart Sass, it is a problem in AdminLTE.

Please see this [PR](https://github.com/twbs/bootstrap/pull/34571) which was released in Bootstrap 4.6.1

@REJack you should consider reopening this issue as it is a real bug in AdminLTE.

**Comment by forxer** [comment]
I sent a PR #4123 to resolve some concerns. 

Hope this helps  :)

**Comment by danny007in** [comment]
just add change configuration in dart sass to not show deprecation warning, @forxer @fritzyfinance

**Comment by forxer** [comment]
I don't want to hide deprecation messages, it's useful. The only solution is to fix the problem

**Comment by REJack** [comment]
Fixed and merged with 0c983d56230a7d1da42a2e46f5a2be6125a1a1d1
